modules/web-console/web-console-server/pom.xml

```diff
@@ -162,6 +162,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-test</artifactId>
```

Comment:
Why we need this dependency?

Reply:
For this amazing annotation @WithMockUser :)